albumentations/augmentations/geometric/transforms.py

```diff
@@ -2038,3 +2039,146 @@ def get_params(self) -> dict[str, D4Type]:
 
     def get_transform_init_args_names(self) -> tuple[()]:
         return ()
+
+class GridElasticDeform(DualTransform):
```

Comment:
You need to add InitPartams class to the tranform that checks input the validity of parameters.

Magintude is non negative,
width and height are > 1